File: python-2569/student/task10.py
n1 = int(input("Enter your number:- "))
n2 = int(input("Enter your number:- "))
n3 = int(input("Enter your number:- "))

if(n1>n2):
    if n1>n3:
        print(n1,"is max")
    else:
        print(n3,"is max")


else:
    if(n2>n3):
        print(n2,"is max")
    else:
        print(n3,"is max")

File: c-language-4820/student/task10.c
#include<stdio.h>

void count(int c)
{
    if (c == 0)
    {
        return;
    }
    printf("The count: %d\n",c--);
    count(c);
}
void main()
{
    int n;

    printf("Enter the number:- ");
    scanf("%d",&n);

    count(n);
}